pipeline/pipeline_tasks/queries/stg_analytics/sba_sfdo_api_calls.py

```diff
@@ -71,18 +85,34 @@ def get_congressional_districts(dbm):
     pass
 
 
+
 def main():
     """Execute Stuff"""
-    print('Getting Yelp data from Yelp API')
     args = get_args()
     dbm = DBManager(db_url=args.db_url)
 
-    sfdo_yelp = get_yelp_fields(dbm)
+    columns = [[
+        'sba_sfdo_id',
+        'borr_name',
+        'borr_street',
+        'borr_city',
+        'borr_state',
+        'borr_zip',
+        'full_address'
+    ]]
+
+    # print('Getting Yelp data from Yelp API')
+    # sfdo_yelp = get_yelp_fields(dbm)
+
+    # print('Getting Geocoded fields from Google Maps API')
+    sfdo_geocoded = get_geocoded_fields(dbm)
+    print(sfdo_geocoded)
+    # sfdo_combined = sfdo_yelp.merge(sfdo_geocoded, on=columns)
```

Comment:
Can we avoid the multiple `dbm.load_table('sba_sfdo', 'stg_analytics')` calls and the merge call here by loading the contents of `sba_sfdo` into a dataframe once, then having get_yelp_fields and get_geocoded_fields each take a dataframe as input and return a dataframe with new columns added?  So you could do something like:
```
sfdo = dbm.load_table('sba_sfdo', 'stg_analytics')
sfdo = get_yelp_fields(sfdo)
sfdo = get_geocoded_fields(sfdo)
dbm.write_df_table(sfdo, table_name='sba_sfdo_api_calls', schema='stg_analytics')
```